subsys/net/lib/softap_wifi_provision/softap_wifi_provision.c

```diff
@@ -798,22 +798,24 @@ static void reset_entry(void *o)
 static const struct smf_state state[] = {
 	[STATE_INIT]	      = SMF_CREATE_STATE(init_entry,
 						 init_run,
-						 NULL),
+						 NULL, NULl, NULl),
```

Comment:
Argh. Sorry. I was reviewing as you merged, just noticed this.

It seems these NULLs have a lower-case L

Reply:
How in the world CI didn't catch this... Will fix